server.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
Interesting post... I somewhat agree with the fact of adding more dependencies for complex ones that requires argument parsing or similar things, but for simple ones (like this one) it only add an extra file, so I don't find it a problem at all.